Dockerfile

```diff
@@ -5,8 +5,11 @@
 FROM ubuntu:xenial-20171114
 LABEL maintainer="Alexander Trost <galexrt@googlemail.com>"
 
-ENV ZULIP_GIT_URL="https://github.com/zulip/zulip.git" \
-    ZULIP_GIT_REF="master"
+ARG ZULIP_GIT_URL=https://github.com/zulip/zulip.git
+ARG ZULIP_GIT_REF=master
+ARG CUSTOM_CA_CERTIFICATES=
+
+SHELL ["/bin/sh", "-xc"]
```

Comment:
This SHELL change should also be its own commit, with a commit message explaining why it's an improvement.